app/components/github_integration/webhooks/issues.py

```diff
@@ -126,26 +128,41 @@ async def _(event: events.IssuesReopened) -> None:
 
     @webhook.event.issues.edited
     async def _(event: events.IssuesEdited) -> None:
-        issue, changes = event.issue, event.changes
+        issue, old_issue_changes = event.issue, event.changes
 
         if issue.created_at > dt.datetime.now(tz=dt.UTC) - dt.timedelta(minutes=15):
             return
 
-        update_notes: list[str] = []
-        if changes.title:
-            update_notes.append(
-                f'Renamed from "{changes.title.from_}" to "{issue.title}"'
+        if old_issue_changes.body and old_issue_changes.body.from_:
+            # Escape code block to avoid breaking diff
+            from_file = old_issue_changes.body.from_.replace("`", r"\`").splitlines(
```

Comment:
Single backquotes don't break the diff, it would be nice if those didn't also have backslashes before them.

```suggestion
            from_file = old_issue_changes.body.from_.replace("```", r"\`\`\`").splitlines(
```

(Note: the comment above will make this unnecessary.)

Reply:
I tried the triple, but if they somehow send \`\`\`\`\` it still breaks. Now, to be fair, this is so highly unlickely. But technically possible, and why I didn't go with it in the first place. 

`````py
print("it does still render properly on GH")
`````